ublue-update.spec

```diff
@@ -56,6 +53,13 @@ flake8 files/etc/%{NAME}.d/system/*.py
 %pyproject_save_files ublue_update
 cp -rp files/etc files/usr %{buildroot}
 
+%pre
+if [ ! -x /usr/bin/topgrade ]
+then
+    echo "Topgrade not installed. Please install Topgrade (https://github.com/topgrade-rs/topgrade) to use %{name}."
```

Comment:
Might be good to build an RPM spec or something as a dep for this package, we could it up in the `bling` COPR or use the `ublue-update` COPR

Reply:
I'll look into it, right now we install with pip. Packaging it may increase our maintenance burden unless we can get upstream to accept a PR

Reply:
Ah, makes sense, i'll resolve this, if we need to revisit making an RPM for topgrade we can do so later on